test/tap/groups/rand10_2/env.sh

```
@@ -0,0 +1,2 @@
+export TEST_PY_TAP_SHUFFLE_LIMIT=10
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)